src/create_disk.sh

```diff
@@ -232,10 +268,18 @@ search --label boot --set prefix
 set prefix=($prefix)/grub2
 normal
 EOF
-
-	# copy the grub config and any other files we might need
-	cp $grub_script rootfs/boot/grub2/grub.cfg
-else
+    mkdir -p rootfs/boot/grub2
+    # copy the grub config and any other files we might need
+    cp $grub_script rootfs/boot/grub2/grub.cfg
+    ;;
+ppc64le)
+    # to populate PReP Boot, i.e. support pseries
+    grub2-install --target=powerpc-ieee1275 --boot-directory rootfs/boot --no-nvram "${disk}${PREPPN}"
```

Comment:
What's the plan wrt petitboot?

Reply:
For petitboot, recent bare metal ppcs(PowerNV), just the grub.cfg on /boot/grub2 should be good enough(no need for PReP Boot partition, etc.) as it searches for know configs on its own.

Reply:
Fantastic. Do we have a way to test this? Shouldn't block this PR, but it would be good to verify that works.

Reply:
I do plan to and will keep you updated.